src/main/java/com/iridium/iridiumskyblock/commands/DepositCommand.java

```diff
@@ -47,7 +47,13 @@ public void execute(CommandSender sender, String[] args) {
         if (island.isPresent()) {
             Optional<BankItem> bankItem = IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getName().equalsIgnoreCase(args[1])).findFirst();
             if (bankItem.isPresent()) {
-                double amount = bankItem.get().deposit(player, Double.parseDouble(args[2]));
+                double amount;
+                try {
+                    amount = bankItem.get().deposit(player, Double.parseDouble(args[2]));
+                } catch (NumberFormatException e) {
```

Comment:
I don't like this short variable name but if you don't care it's fine, I guess

Reply:
I did not name it

Reply:
I know, IntelliJ chooses e by default, just rename it 👀